### ISSUE TITLE
Fix samples readme reference

### ISSUE THREAD

**Issue by jsoref** [googleapis/synthtool#431]
Thanks for stopping by to let us know something could be better!

#### Environment details

  - `synthtool` version: master

#### Steps to reproduce

  1. Read the sentence in https://github.com/googleapis/nodejs-dlp/blob/master/README.md#samples
  2. Determine that the grammar is confusing and that the message is wrong.
  3. File https://github.com/googleapis/nodejs-dlp/pull/423
  4. Told [to file an issue](https://github.com/googleapis/nodejs-dlp/pull/423#issuecomment-596765256)

**Comment by jsoref** [comment]
@bcoe:
Afaiu, you're thinking about https://github.com/googleapis/synthtool/blob/master/synthtool/gcp/templates/node_library/samples/README.md which is a separate file.

If you mean that the message itself is talking about those files, then the change should be of the form:
```patch
## Samples

-Samples are in the [`samples/`](https://github.com/{{  metadata['repo']['repo'] }}/tree/master/samples) directory. The samples' `README.md`
-has instructions for running the samples.
+Samples are in the [`samples/`](https://github.com/{{  metadata['repo']['repo'] }}/tree/master/samples) directory. Each sample's `README.md`
+has instructions for running its sample.
```

I'm happy to make a PR for that if that's the intention. Or if the intention is closer to my initial interpretation, I'm happy to make a PR for that.

**Comment by sofisl** [comment]
Addressed in #872, will hold off on merging until after fix-it week
